### PR TITLE
Add open source statement on landing page

### PR DESCRIPTION
**Pull request by mit-mit**
Fixes https://github.com/dart-lang/site-www/issues/3158

Staged here: https://mit-staging.web.app/

Screenshot 2021-04-28 at 15 28 25

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -35,6 +35,13 @@
     </div>
 
     <img class="made-by" src="assets/dash/2x/made by google@2x.png"/>
+    <div class="callout-oss">
+        <p>Dart is free, and open source
```
Comment (kwalrath) [inline src/index.html:39]:
```suggestion
        <p>Dart is free and open source
```

I was unsure about the comma, so I checked the Flutter page. It doesn't have one ("It’s free and open source."), and we should probably follow its lead.

**Review by kwalrath — COMMENTED** [review]
One nit, assuming the rest is OK with everyone.

File: src/index.html (hunk 1 above)
Comment (mit-mit) [reply src/index.html:39]: Sounds good.

Should there be a hyphen in 'open source', like: https://en.wikipedia.org/wiki/Free_and_open-source_software ?

Comment (parlough) [reply src/index.html:39]: Based on my experiences, people will likely be more familiar with "open source" without a dash.

Comment (kwalrath) [reply src/index.html:39]: No hyphen, but it's a good question. Google has a policy of not using a hyphen for "open source", even when  it's a verb or it precedes a noun (which this one doesn't): https://developers.google.com/style/word-list#letter-o.

**Review by timsneath — APPROVED** [review]

**Review by parlough — APPROVED** [review]

**Pull request merged**